src/GeoPing.FrontEnd/src/state/allUsersTableState.ts

```diff
@@ -0,0 +1,145 @@
+import IAllUsersTableStateType from '../DTO/types/stateTypes/allUsersTableStateType';
```

Comment:
ahh... you percept mocked up table literally ))). no there shuld be another columns, for now username, email, mail, last login, number of lists, number of shared lists.

Reply:
I just do not know what should be there, took the field from the layout, correct it